admin/server/middleware/browserify.js

```diff
@@ -35,7 +36,16 @@ module.exports = function (file, name) {
 	var logName = typeof file === 'string' ? file.replace(/^\.\//, '') : name;
 	var fileName = logName;
 	if (fileName.substr(-3) !== '.js') fileName += '.js';
+
+	var writeToDisk = keystone.get('write keystone files to disk');
+	var staticPath = keystone.get('static') || path.join(keystone.get('module root'), 'public');
+	var outputPath = path.resolve(path.join(staticPath, 'keystone', fileName));
+
 	function writeBundle (buff) {
+		if (writeToDisk) {
+			fs.outputFile(outputPath, buff, 'utf8');
```

Comment:
doesn't need to use a callback since the value of the buffer is attached to the closure variable `src` already by this point, but this still makes me uncomfortable. I'll add a cb just to check the error value since it happens during production, not just dev.